functions_complete.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May 13 13:16:43 2020
+
+@author: carolinstreitberger
+"""
+######## Imports
+import numpy as np
+import random
+
+######## The first step is to create the grid of the computer
+def generate_random_position(size_ship):
```

Comment:
While I haven't seen you make a mistake in this code, I think it's asking for trouble to have such similarly named variables: size_ship and size_ships.
Especially, since both are so prominent in your code (one appears 12 times, the other 22 times), you could make mistakes this way (and also make it more confusing for "readers" looking over your code).

How about something like current_ship_size (for size_ship) or all_ship_sizes (for size_ships) or even just size_ship_1 and size_ship_2?